build/bakefiles/common.bkl

```diff
@@ -2,7 +2,7 @@
 
 <makefile>
 
-    <requires version="0.2.7"/>
+    <requires version="0.2.10"/>
```

Comment:
Is this really necessary? Not that it hurts anyhow, of course, but I just wonder why was this changed.

Reply:
The reason for the two changes above, is that there actually is no bakefile 0.2.11, the latest version of the legacy branch is 0.2.10, and the version check is broken so bakefile thinks that 0.2.10 is 0.2.1 and is below 0.2.7.

Reply:
This is indeed annoying and confusing (sorry...), but while the 0.2.11 release hasn't been done yet, the [latest branch](https://github.com/vslavik/bakefile/tree/legacy-0.2-branch) definitely does use 0.2.11.

Reply:
Ah ok, I'll change that back then and use the git version.